### PR TITLE
add how to give proofreader permissions on crowdin

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This is a question that often appear between language leads, so now it's in the docs

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]

File: docs/language-lead-handbook.md [hunk 1]
```diff
@@ -289,3 +289,18 @@ Then, each target language has two columns. If you translate to Dothraki, you wi
 > In programs like Microsoft Excel you can hide the columns of the other languages to free up screen real-estate and see the English columns and the target language columns near each other.
 
 After you have made the changes and saved the file, you will need to make a PR with the proposed changes. After the PR is accepted, you will need to run the GitHub Action workflow to update the Crowdin Glossary. Your glossary changes will not have immediate effects, but they will come.
+
+## How to Promote a Contributor to Proofreader
+
+If you consider that a contributor could become a Crowdin Proofreader, you can give the proofreader role to them in this way:
+
+In crowdin, individuate the `User management" on the left handside menu.
```
Comment (Sboonny) [inline docs/language-lead-handbook.md:297]:
```suggestion
In Crowdin, individuate the `User management` on the left hand side menu.
```

**Review by Sboonny — COMMENTED** [review]
Small nitpicks

**Review by naomi-lgbt — APPROVED** [review]
banger pr, thanks for contributing!

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**